q2_demux/_summarize/assets/overview.html

```diff
@@ -25,9 +25,28 @@ <h1>Demultiplexed sequence counts summary</h1>
 </div>
 {% endif %}
 
+<div class="row">
+    <div class="col-lg-12">
+        <h1>Demultiplexed sequence length summary</h1>
+        {% set colsize = 6 if reverse_length_table else 12 %}
+        <div class="col-lg-{{ colsize }}">
+            <h4>Forward Reads</h4>
+            {{ forward_length_table }}
+        </div>
+        {% if reverse_length_table %}
+        <div class="col-lg-{{ colsize }}">
+            <h4>Reverse Reads</h4>
+            {{ reverse_length_table }}
+        </div>
+        {% endif %}
+    </div>
+</div>
+
+
 <div class="row">
     <div class="col-lg-12">
         <h1>Per-sample sequence counts</h1>
+        <h4>Total Samples: {{ n_samples }}</h4>
```

Comment:
Can we also add the total reads (before subsampling)?

Reply:
Would this be relocating or in addition to the first table's `Total` row?

Reply:
I am not a smart man. I think I got confused while reviewing - please disregard!